bigquery_datatransfer/nox.py

```diff
@@ -13,24 +13,54 @@
 # limitations under the License.
 
 from __future__ import absolute_import
+
 import os
 
 import nox
 
 
+LOCAL_DEPS = (
+    os.path.join('..', 'api_core'),
+    os.path.join('..', 'core'),
+)
+
+
 @nox.session
-@nox.parametrize('python_version', ['2.7', '3.4', '3.5', '3.6'])
-def unit_tests(session, python_version):
-    """Run the unit test suite."""
+def default(session):
+    """Default unit test session.
+
+    This is intended to be run **without** an interpreter set, so
+    that the current ``python`` (on the ``PATH``) or the version of
+    Python corresponding to the ``nox`` binary the ``PATH`` can
+    run the tests.
```

Comment:
This docstring reminds of why `default` exists to begin with: for running on AppVeyor.

@chemelnucfin Can you add the 3 new APIs to `appveyor.yml` as well?